onos-config/templates/_helpers.tpl

```diff
@@ -117,3 +117,16 @@ openpolicyagent image name
 {{- .Values.openpolicyagent.image.tag -}}
 {{- end -}}
 {{- end -}}
+
+{{/*
+  modelPlugins image names,
+  takes the image name and the .Values to add the registry if any.
+*/}}
+{{- define "modelplugin.imagename" -}}
+{{- if .Values.global.image.registry -}}
```

Comment:
Is it ok to use the same registry as the `onos-config` image or do we want to specify a different one? I can't really imagine a usecase in which images come from a different registry, but who knows...

Reply:
Hmmm... I think for now, let's use the same registry. If the usecase emerges for supporting different ones, we'll cross that bridge then.

Reply:
Sounds good